resources.yaml

```diff
@@ -201,3 +195,9 @@ deployment:
         start: 2022-01-12
         end: 2022-01-12
         group: training-imbs2022
+    training-nhm-:
```

Comment:
is the leading `-` intendet?

Reply:
If you mean the last one, those (nhm-) are the 4 first characters of the id provided by the trainer (nhm-postgrad). We've had cases like this in the past and it has worked.

Reply:
ok, cool!